accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -701,8 +701,8 @@ public void markShardDurable(SafeCommandStore safeStore, TxnId globalSyncId, Ran
     protected void updatedRedundantBefore(SafeCommandStore safeStore, TxnId syncId, Ranges ranges)
     {
         TxnId clearWaitingBefore = redundantBefore.minShardAndLocallyAppliedBefore();
-        TxnId clearAnyBefore = durableBefore().min.majorityBefore;
-        progressLog.clearBefore(safeStore, clearWaitingBefore, clearAnyBefore);
+        TxnId clearAllBefore = TxnId.min(clearWaitingBefore, durableBefore().min.majorityBefore);
```

Comment:
why push this change here?  it looked to happen in default progress log before now its an assert and the logic moved here.

Reply:
It’s clearer to have the caller declare what they actually want rather than have the callee reinterpret, which can be surprising